### PR TITLE
Board of Recommendation

### PR DESCRIPTION
**Pull request by royklip**
Add the Board of Recommendation between the information page card and the location page card.

### CODE REVIEW

**Comment by TimvdLippe** [comment]
You need to rebase this PR, because some commits should not be in this diff.

**Comment by martijnjanssen** [comment]
I think this is because the old one was merged into the new one that contained my commit.

**Comment by martijnjanssen** [comment]
It might be nice to add a minimum width to the company logos, currently, the distribution of the logos is not very even. Other than that, looks good!

File: src/symposium-app/symposium-recommendations.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                border-top: 1px rgba(128,128,128,0.3) solid;
+            }
+
+            .company {
+                flex: 1;
+                flex-basis: 150px;
+                height: 250px;
+                padding: 10px;
+                text-align: center;
+                margin: 0 5px 0 5px;
```
Comment (martijnjanssen) [inline src/symposium-app/symposium-recommendations.html]: You can remove the padding and add those `10px` to the margin: `margin: 10px 15px;`

File: src/symposium-app/symposium-recommendations.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                vertical-align: middle;
+            }
+
+            .recommendations {
+                border-top: 1px rgba(128,128,128,0.3) solid;
+            }
+
+            .company {
+                flex: 1;
+                flex-basis: 150px;
```
Comment (martijnjanssen) [inline src/symposium-app/symposium-recommendations.html]: I would suggest increasing the flex basis to `175px`, this improves the formatting of the text underneath the images.

**Review by martijnjanssen — CHANGES_REQUESTED** [review]
Changes all look very good, just two really small remarks and we are good to go!

**Review by martijnjanssen — APPROVED** [review]

**Comment by martijnjanssen** [comment]
All looks good to me!

**Pull request merged**